x/auth/migrations/legacytx/stdtx.go

```diff
@@ -107,66 +102,13 @@ func NewStdTx(msgs []sdk.Msg, fee StdFee, sigs []StdSignature, memo string) StdT
 // GetMsgs returns the all the transaction's messages.
 func (tx StdTx) GetMsgs() []sdk.Msg { return tx.Msgs }
```

Comment:
Can we remove `StdTx` too? Afaik the only thing we need to keep in this package is StdSignDoc

Reply:
Unfortunately no. Turns out we added support back into 0.47: https://github.com/cosmos/cosmos-sdk/issues/10856